SofaKernel/modules/SofaBaseCollision/SofaBaseCollision_test/DefaultPipeline_test.cpp

```diff
@@ -25,6 +25,9 @@ using std::vector;
 #include <string>
 using std::string;
 
+#include <sofa/core/fwd.h>
+using sofa::core::execparams::defaultInstance; /// From sofa/core/fwd.h
```

Comment:
This Doxygen comment has no effect.

Reply:
Actually I use /// to indicates code's comment that should stay in the code base from 
temporary comments with //

I'm not sure it makes sense to expect a doxygen comment here.